sycl/test/check_device_code/cuda/matrix/matrix-nvptx-compile-query-test.cpp

```diff
@@ -0,0 +1,33 @@
+// REQUIRES: cuda
```

Comment:
Can you update `CODEOWNERS` for the new cuda/matrix and hip/matrix directories?

Reply:
I added `intel/llvm-reviewers-cuda` to be the owner of those directories